front/src/applications/editor/tools/pointEdition/components.tsx

```diff
@@ -230,8 +231,52 @@ export const PointEditionLeftPanel: FC<{ type: EditoastType }> = <Entity extends
       )}
       <EditorForm
         data={state.entity as Entity}
+        overrideUiSchema={{
+          logical_signals: {
+            items: {
+              signaling_system: {
+                'ui:widget': 'hidden',
+              },
+              settings: {
+                'ui:description': ' ',
+                Nf: {
+                  'ui:description': ' ',
+                  'ui:widget': CustomFlagSignalCheckbox,
+                },
+                distant: {
+                  'ui:description': ' ',
+                  'ui:widget': CustomFlagSignalCheckbox,
+                },
+                is_430: {
+                  'ui:description': ' ',
+                  'ui:widget': CustomFlagSignalCheckbox,
+                },
+              },
+            },
+          },
+        }}
         onSubmit={async (savedEntity) => {
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          const checkedSavedEntity = () => {
+            if (
+              savedEntity.properties.logical_signals &&
+              !savedEntity.properties.logical_signals[0].signaling_system
+            ) {
+              return {
+                ...savedEntity,
+                properties: {
+                  ...savedEntity.properties,
+                  logical_signals: [
+                    {
+                      ...savedEntity.properties.logical_signals[0],
+                      signaling_system: 'BAL',
+                    },
+                  ],
+                },
+              };
+            }
+            return savedEntity;
+          };
```

Comment:
When we check one of these options and save the form, it seems that the change is not taken into account.

Reply:
The change is saved, but i forgot to give a `checked` attribute to the checkbox to match the value received... Thanks :)